frontend/src/components/pages/RegistrantExperience/RegistrationSteps/index.tsx

```diff
@@ -38,10 +39,18 @@ const RegistrationSteps = ({
   >(waiverReducer, {
     campName: camp.name,
     waiver,
-    optionalClauses: [],
-    requiredClauses: [],
+    optionalClauses: waiver.clauses.reduce(
```

Comment:
would filter + map be a cleaner solution here?

Reply:
Eh reduce can be read in one go, and only requires one iteration